src/Dapr.Actors/Client/ActorProxyFactory.cs

```diff
@@ -55,22 +55,26 @@ public TActorInterface CreateActorProxy<TActorInterface>(ActorId actorId, string
         /// <inheritdoc/>
         public ActorProxy Create(ActorId actorId, string actorType, ActorProxyOptions options = null)
         {
+            options ??= this.DefaultOptions;
```

Comment:
This was just plain wrong. Passing in a new "options" is supposed to win over the defaults.